testing/munin/munin-node.post-install

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# create directories
+mkdir -p /usr/share/munin/docs
+mkdir -p /var/lib/munin/spool
+mkdir -p /var/log/munin
+mkdir -p /var/run/munin
```

Comment:
`/var/run/munin` is a symlink to `/run` that is a tmpfs, i.e. it’s wiped on every restart. Therefore it doesn’t make any sense to create it here, moreover you already creates it in runscript.